addons/apecloud-mysql/templates/cmpd-apecloudmysql.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: {{ include "apecloud-mysql.componentDefName" . }}
```

Comment:
Does apecloud-mysql need to support multiple versions? If multi-version support is required, it is recommended to reconsider the naming of cmpd.

Reply:
it only supports mysql 8 (major version). so we keep it as it is.